model/src/main/scala/org/broadinstitute/dsde/rawls/model/RawlsEnumeration.scala

```diff
@@ -1,6 +1,6 @@
 package org.broadinstitute.dsde.rawls.model
 
-trait RawlsEnumeration[T <: RawlsEnumeration[T]] { self: T =>
+trait RawlsEnumeration[T <: RawlsEnumeration[T]] extends Product with Serializable { self: T =>
```

Comment:
Another explanation, including why one should use final as well: http://underscore.io/blog/posts/2015/06/04/more-on-sealed.html